projects/webrender-release-66/index.html

```diff
@@ -0,0 +1,660 @@
+<!DOCTYPE html>
```

Comment:
Is this file rendered from some source file or did you compile this doc by hand? If it's compiled, can you include your source material as well?

Reply:
Included! Up to a nice round 100 files